lib/AccessibilitySupport/AccessibilitySupport.js

```diff
@@ -100,6 +117,10 @@ module.exports = {
 		var prefix = this.accessibilityLabel || this.content || null;
 		var label;
 
+		if (this.accessibilityDisabled) {
```

Comment:
This is a bit a personal preference but I'd rather see these checks as one line without the braces. No need for the extra lines of code for a kick out.

```
if (this.accessibilityDisabled) return;
```